### DISCUSSION TITLE
Image compression on upload

### DISCUSSION THREAD

**Discussion by lindamacdonald** [rpldy/react-uploady#211] [General]
Hey there guys :) would be fabulous to have image compression on uploaded images to make our apps as performant as possible. I have only been able to find one way to do this but as you can see this isnt really a specific option for react:

https://www.jpegmini.com/developers

images being the number 1 culprit when it comes to load performance it is really good to have images compressed. Of course if its our own app then we can manually compress beforehand but if you have a saas you cannot expect clients alway to  be able to manage compression

**Comment by yoavniran** [comment]
Hey @lindamacdonald 
Media performance is a topic that is dear to my heart and close to home as I happen to work for [Cloudinary](https://cloudinary.com). Its a service that offers amazing tools and capabilities around compression, performance, and so much more...

However, this isnt an Uploady issue and I dont think compression should be done on the client/upload side. I think using a service like Cloudinary makes a lot of sense since you get to keep the original, full resolution asset but then serve it compressed/optimized to your users as needed.

Im moving this to the discussions section in case you wish to discuss more.
